packages/code-editor/src/customTabs/Markdown/index.js

```diff
@@ -83,6 +112,7 @@ export default class Markdown extends Component {
           top: '1.25rem',
           zIndex: 10
         }}>
+          {this.renderTogglePublicButton()}
```

Comment:
Please check if the switch button should be displayed in the other `.md` file (not `README.md`)，if only display in the 'README.md', can improve it in another PR.

![image](https://user-images.githubusercontent.com/20736452/151663563-bd165dd3-8e9a-49f7-9ed2-395257cc1852.png)